rdt/hyper_transformer.py

```diff
@@ -370,10 +370,6 @@ def _fit_field_transformer(self, data, field, transformer):
                 if self._field_in_data(output_field, data):
                     self._fit_field_transformer(data, output_field, next_transformer)
 
-            else:
-                if output_name not in self._output_columns:
-                    self._output_columns.append(output_name)
```

Comment:
Deleting this because we need to reset the `_output_columns` when sorting them (this also doesn't get used before the transform/reverse_transform, so moving it later in the fit doesn't affect anything).